src/__tests__/testcases/generics/index.ts

```diff
@@ -30,3 +31,5 @@ export class Cl<T = E> {
 export function fn<T = G>(g: T, h: Gen<H>) {}
 export type TyFn = <T = J>(j: T, k: Gen<K>) => L;
 export type TyCtor = new <T = M>(m: T, n: Gen<N>) => O;
+export interface I2 extends Gen<P> {
```

Comment:
Nice!

I recently implemented type parameter shadowing.
Could you add this case to that test as well?

https://github.com/Swatinem/rollup-plugin-dts/blob/a831c2554330a4eb5846c0634a28f9bda2e58cbb/src/__tests__/testcases/shadowing/exports.ts#L19-L22

Reply:
I'm not 100% about how those shadowing tests work. Is this right?
```typescript
export interface GenericExtends<M = any, N = M> extends GenericInterface<M, N> {}
```

Reply:
I've added (what I think is) a test for this in bed58e741f4413ea08ce6732c5e20ff2dac42228. If that's not correct, let me know. 🙂